java/src/jmri/Block.java

```diff
@@ -680,9 +680,6 @@ public void goingActive() {
                 count++;
                 next = pList[i];
             }
-            log.debug("\n{} is {}\n{} is {}",
-                    Path.decodeDirection(pDir[i]), String.format("%16s", Integer.toBinaryString(pDir[i])).replace(' ', '0'),
```

Comment:
You can also push things like this down to the "trace" level with `log.trace(...)`.  That's below debug, so debug can be turned on without those showing, or trace can be turned on to get even more info.